templates/zerver/about.html

```diff
@@ -0,0 +1,124 @@
+  Zulip has an extensive real-time messaging featureset, including
+  both group and private messaging, conversation streams, powerful
+  search, drag-and-drop file uploads, image previews, audible
+  notifications, missed-message emails, markdown formatting, desktop
+  and mobile apps, dozens of integrations, and much, much more.
+</p>
+
+<p>
+  Zulip was designed from the ground up to optimize the productivity
+  of discussions and real-time decision-making.  A key feature that
+  drives this is Zulip's unique threading model and system for
+  tracking unread messages, which makes it easy to have multiple
+  simultaneous conversations in the same stream.  As a result, Zulip
+  is more efficient than any other chat product for catching up on
+  conversations you missed while you were away from your devices.  You
+  can read exactly the threads that are important to you, and it feels
+  natural to follow up on conversations that happened while you were
+  away.
+</p>
+
+<p>
+  Zulip's vision is to create the world's best group chat software,
+  completely open source, so that everyone has the freedom to
+  customize, improve, and run their own copy of this essential piece
+  of collaboration infrastructure.
+</p>
+
+<p>
+  Further information on the Zulip project and <a href="/features">its
+  features</a> can be found on
+  the <a href="https://www.zulip.org">Zulip open source project's
+  website</a> and
+  the <a href="http://zulip.readthedocs.io/en/latest/">Zulip
+  documentation</a>.
+</p>
+
+<h2>Zulip Community</h2>
+
+<p>
+  Zulip is developed by a vibrant community of developers from around
+  the world.  Every month, dozens of people contribute code to Zulip,
+  and dozens more contribute bug reports, feedback, and translations.
+  The Zulip community welcomes new contributors from any background.
+  The Zulip project has an easy to install development environment, an
+  extensive test suite,
+  and <a href="http://zulip.readthedocs.io/en/latest/">over 50,000
+  words of developer documentation</a> to make it easy for new
+  contributors to contribute effectively to the project.
+</p>
+
+<h3>Contributing to Zulip</h3>
+
+<p>
+  If you'd like to join the Zulip community and contribute to Zulip,
+  we'd love to have you!  Please
+  visit <a href="https://github.com/zulip/zulip/#zulip-overview">the
+  main Zulip project on GitHub</a> for details on how to get involved!
+</p>
+
+<h2>Zulip history</h2>
+
+<p>
+  Zulip was originally developed by Zulip, Inc., a small startup in
+  Cambridge, Massachusetts.  Zulip, Inc. was founded by the MIT team
+  that previously created
+  <a href="https://www.ksplice.com">Ksplice</a>, software for
+  live-patching a running Linux kernel.  Zulip was inspired by
+  the <a href="https://barnowl.mit.edu/">Barnowl</a> client for
+  the <a href="https://en.wikipedia.org/wiki/Zephyr_(protocol)">Zephyr</a>
+  protocol, and the incredible community that Zephyr supported at MIT.
```

Comment:
Suggest:

> The application was originally developed by Zulip, Inc., a small startup in Cambridge, Massachusetts.  The startup was founded by the MIT team that had previously created <a href="https://www.ksplice.com">Ksplice</a>, software for live-patching a running Linux kernel.  Zulip itself was inspired by the <a href="https://barnowl.mit.edu/">Barnowl</a> client for the <a href="https://en.wikipedia.org/wiki/Zephyr_(protocol)">Zephyr</a> protocol, and the incredible community that Zephyr supported at MIT.

Also, no mention of Humbug? I think at least a passing cameo is appropriate in the first line.